index.js

```diff
@@ -381,7 +375,7 @@ export class Placement {
    * string from API response. Can also be null, indicating a noop action.
    */
   fetch() {
-    const callback = "ad_" + Date.now();
+    const callback = "ad_" + Date.now() + "_" + Math.floor(Math.random() * 1000000);
```

Comment:
Might be worth a comment -- I assume this is to avoid conflicts w/ multiple placements?